multicluster/Makefile

```diff
@@ -50,11 +50,13 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
-ENVTEST_ASSETS_DIR=$(shell pwd)/testbin
```

Comment:
Why these are no longer needed?

Reply:
this actually doesn't work, we will add our own codes and scripts to start a new env, run integration test etc